perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/AuthzResolverMethod.java

```diff
@@ -96,18 +96,12 @@ public AuthzRoles call(ApiCaller ac, Deserializer parms) throws PerunException {
 		public List<RichUser> call(ApiCaller ac, Deserializer parms) throws PerunException {
 			//get role by name
 			String roleName = parms.readString("role");
-			Role role;
-			try {
-				role = Role.valueOf(roleName);
```

Comment:
You should check if the role exists in the database.

Reply:
Done